creator-node/src/routes/tracks.js

```diff
@@ -684,7 +685,7 @@ module.exports = function (app) {
           }
         }
       } catch (e) {
-        req.logger.error(`Error looking for stream fallback in redis`, e)
+        req.logger.error('Error looking for stream fallback in redis', e)
```

Comment:
im not sure if the bunyan logger can handle printing objects https://github.com/trentm/node-bunyan#log-method-api

we might want to do sometihng like this
```
log.info({foo: 'bar', err: err}, 'some msg about this error');
                // To pass in an Error *and* other fields, use the `err`
                // field name for the Error instance **and ensure your logger
                // has a `err` serializer.** One way to ensure the latter is:
                //      var log = bunyan.createLogger({
                //          ...,
                //          serializers: bunyan.stdSerializers
                //      });
                // See the "Serializers" section below for details.
```